src/api/lib/tasks/dev.rake

```diff
@@ -159,8 +159,8 @@ namespace :dev do
           source_package: leap_apache
         )
         new_package1.staging_project = sworkflow.staging_projects.first
-        new_package1.reviews << create(:review, by_project: new_package1.staging_project)
         new_package1.save
+        create(:review, by_project: new_package1.staging_project, bs_request: new_package1)
```

Comment:
@eduardoj why is this called `new_package1` if it is actually a request? confusing... but not introduced by you :)

Reply:
@krauselukas, these tests were introduced in commit feb8cd2 by @coolo. We can always ask him.